pennylane_lightning/core/src/utils/cuda_utils/cuda_helpers.hpp

```diff
@@ -151,6 +151,22 @@ inline static constexpr auto complexToCu(CFP_t a) {
     }
 }
 
+/**
+ * @brief Utility to convert a vector of std::complex types to cuComplex types
+ *
+ * @tparam CFP_t std::complex types.
+ * @param vec A std::vector<std::complex> type.
+ * @return a vector of cuComplex converted vec
+ */
+template <class CFP_t = std::complex<double>>
+inline auto complexToCu(const std::vector<CFP_t> &vec) {
```

Comment:
Could we use this to get rid of cuGates_host.hpp now? (maybe with some transpose?)

Reply:
yea, we can. But let's come back to refactor it later since it's not a priority for now.